src/jcsclient/utils.py

```diff
@@ -158,6 +158,44 @@ def dash_to_camelcase(keyword):
         keyword += part
     return keyword
 
+def push_monitoring_indexed_params(params, key, vals):
```

Comment:
yes anant see carefully, there is a change in parsing of data below, this does not remove the trailing 's' and adds a .member.<number>